tests/functional/vault/test_withdrawal.py

```diff
@@ -247,12 +247,12 @@ def test_withdrawal_with_empty_queue(
     assert vault.balanceOf(gov) == strategy_balance
     assert token.balanceOf(gov) == free_balance
 
-    # Calling it a second time with strategy_balance should be a no-op
-    vault.withdraw(
-        strategy_balance * vault.pricePerShare() // 10 ** vault.decimals(),
-        {"from": gov},
-    )
-    assert token.balanceOf(gov) == free_balance
+    # Calling it a second time with strategy_balance should revert as we cannot withdraw more shares
+    with brownie.reverts():
+        vault.withdraw(
+            strategy_balance * vault.pricePerShare() // 10 ** vault.decimals(),
```

Comment:
this actually should be `strategy_balance * (10 ** vault.decimals()) // vault.pricePerShare()`. I have this PR https://github.com/yearn/yearn-vaults/pull/516 out to fix this.